lib/rules/jsx-first-prop-new-line.js

```diff
@@ -15,14 +15,63 @@ module.exports = {
       category: 'Stylistic Issues',
       recommended: false
     },
+    fixable: 'code',
 
-    schema: [{
-      enum: ['always', 'never', 'multiline']
-    }]
+    schema: [
+      {
+        enum: ['always', 'never', 'multiline']
+      },
+      {
+        oneOf: [{
+          enum: ['tab']
+        }, {
+          type: 'integer'
+        }]
+      }
+    ]
   },
 
   create: function (context) {
     var configuration = context.options[0];
+    var extraColumnStart = 0;
+    var indentType = 'space';
+    var indentSize = 2;
+    var sourceCode = context.getSourceCode();
+
+    if (context.options.length > 1) {
+      if (context.options[1] === 'tab') {
+        indentSize = 1;
+        indentType = 'tab';
+      } else if (typeof context.options[1] === 'number') {
+        indentSize = context.options[1];
+        indentType = 'space';
+      }
+    }
+
+    function getNodeIndent(node, byLastLine, excludeCommas) {
+      byLastLine = byLastLine || false;
```

Comment:
`getNodeIndent` is only called in one place, and called with `byLastLine` and `excludeCommas` set to false. Can just remove these 2 variables and short circuit their values into this function to make it simpler.